build/build.proj

```diff
@@ -164,6 +164,7 @@
           DestinationFolder="$(TestsDirectory)"
           />
 
+<Message Text="$(DotNetTool) &quot;$(TestsDirectory)\xunit.console.netcore.exe&quot; &quot;@(TestAssembly, '&quot; &quot;')&quot; -xml &quot;@(XmlTestFile)&quot;" Importance="High" />
```

Comment:
Did you mean to check this in. I like it, but the formatting is off and it would be nice not to duplicate the command in Message and Exec.